### ISSUE TITLE
Clickhouse auto completition 

### ISSUE THREAD

**Issue by ghuname** [dbeaver/dbeaver#4496]
On version 5.2.3.201810201753, when I write select * from, then I type first few letters (let's say "def"), I get database auto completion list, with `default` in list. 
If I select `default` with keyboard arrows or mouse i get `default`def, so I have to delete def from the right side in order to continue.
The same is with field names. It used to work well in previous dbeaver versions.
I am connected to the Clickhouse database.

**Comment by p91paul** [comment]
This was already reported and 5.2.3 re-released. You need to visit dbeaver.io and download and install 5.2.3 again!

**Comment by ghuname** [comment]
@p91paul 

Well if it is reported and solved, you should increment the version, and offer it through auto update process.

**Comment by p91paul** [comment]
I agree with you, that is why I opened bug #4490. I'm just a user here!

**Comment by serge-rider** [comment]
@ghuname This particular issue was fixed in the very same day of 5.2.3 release. Making two releases in the same day is a bit strange.
So we decided to make a hotfix build. But I agree that we need some special procedure for such cases.
